.github/support.yml

```
@@ -0,0 +1,29 @@
+# Configuration for Support Requests - https://github.com/dessant/support-requests
+
+# Label used to mark issues as support requests
+supportLabel: support
+
+# Comment to post on issues marked as support requests, `{issue-author}` is an
+# optional placeholder. Set to `false` to disable
+supportComment: |
+  Hi there @{issue-author} :wave:!
+
+  I closed this issue because it was labelled as a support question.
+
+  Please repost this on the http://discourse.jupyter.org/ forum (most people who hang out here also hang out there, as well as more people).
+  We are trying to streamline the different discussion places and want to use the issues on GitHub repos for technical discussions on how to change the contents of the repo.
+  More general discussions should go to Discourse so that they are easier to find, better indexed by google and generally more accessible than being hidden in the bowels of a GitHub repository :smiley:
```

```suggestion
  Please repost this on the http://discourse.jupyter.org/ forum to help organize our discussion. The goal is to let issues on GitHub streamline discussions on how to improve its content and let the forum streamline discussion of other things.
```

What do you think about this suggestion? I think it often could be beneficial for an individual to post on GitHub, because it may be a more efficient way to get the attention of active maintainers than on the discourse forum, so it felt a bit deceptive to suggest it would be better for the individual to do this. But, it is a matter of sustainability in my mind. If everyone would follow this practice, I do think it would be better for everyone. Anyone looking in github would have an easier time finding actual bugs and discussions on improvements, and anyone looking for help would have an easier time finding others having asked the same question.

---

I like it :sparkles:.  I would also keep the google indexing part and the emoji. Maybe something like this?

```suggestion
  Please repost this on the http://discourse.jupyter.org/ forum to help organize our discussion.
  The goal is to let issues on GitHub streamline discussions on how to improve the contents of the repo and let the forum streamline discussion of all other things so that they are easier to find, better indexed by Google and generally more accessible than being hidden in the bowels of a GitHub repository :smiley:"
```

WDYT @consideRatio ?

---

:D I also want to have an emoji somewhere in this message!

I have two considerations:
- I'm consider the statement about discourse being better indexed than GitHub. Is it so? I have no subjective experience or objective data about that.
- Is discourse really more accessible?

---

> I'm consider the statement about discourse being better indexed than GitHub. Is it so? I have no subjective experience or objective data about that.

I could only find [this comment](https://github.com/jupyterhub/team-compass/issues/73#issuecomment-433120655) but it's comparing Discourse against  Gitter (the "searcheability" is obviously true in this case :smile:).

Do you want to edit your commit suggestion and only add the emoj then :heart: ?

---

I think that Discourse is more discoverable - for example GitHub issues don't show up in google search index, while Discourse posts do.

---

```suggestion
  Please help us organize discussion by posting this on the http://discourse.jupyter.org/ forum.

  Our goal is to sustain a positive experience for both users and developers. We have planned for GitHub issues to streamline specific discussions on how to improve the GitHub repository's content, and let the forum be where we freely can help and inspire each others.
  
  Thanks you for being an active member of our community! :heart:
```

This is hard! I made another iteration! WDYT?

---

I wasn't sure how to make a PR to your comment, so I edited it directly, I hope that is OK! What do you think?

---

LGTM! What do you think @GeorgianaElena ?

> I think that Discourse is more discoverable - for example GitHub issues don't show up in google search index, while Discourse posts do.

@choldgraf I've experience the opposite, I think, I've only my subjective experience to go on and brief tests of searching for words to verify google can find either if given high enough incentive. Perhaps our search history with google influence this.

---

good point - I'm not too sure. Either way I think that the fact that GitHub issues are "closable" is a challenge from a discoverability perspective, but either way I think there is a time and a place for both!

---

I love it! Thanks @consideRatio, @choldgraf